packages/sync/src/class-actions.php

```diff
@@ -127,11 +127,26 @@ public static function add_sender_shutdown() {
 			self::should_initialize_sender()
 		) ) {
 			self::initialize_sender();
-			add_action( 'shutdown', array( self::$sender, 'do_sync' ) );
-			add_action( 'shutdown', array( self::$sender, 'do_full_sync' ), 9999 );
+			self::hook_sender();
 		}
 	}
 
+	/**
+	 * Send on shutdown.
+	 */
+	public static function hook_sender() {
+		add_action( 'shutdown', array( self::$sender, 'do_sync' ) );
+		add_action( 'shutdown', array( self::$sender, 'do_full_sync' ), 9999 );
+	}
+
+	/**
+	 * Don't Send on shutdown.
+	 */
+	public static function unhook_sender() {
+		remove_action( 'shutdown', array( self::$sender, 'do_sync' ) );
+		remove_action( 'shutdown', array( self::$sender, 'do_full_sync' ), 9999 );
```

Comment:
The 9999 is not necessary here.

Reply:
> To remove a hook, the $function_to_remove and $priority arguments must match when the hook was added. 

https://developer.wordpress.org/reference/functions/remove_filter/

Reply:
Where are we using the `unhook_sender` method?